crates/driver/src/api/execute.rs

```diff
@@ -20,14 +20,21 @@ pub fn post_execute(
 ) -> impl Filter<Extract = (ApiReply,), Error = Rejection> + Clone {
     post_execute_request(prefix).and_then(move |summary: SettlementSummary| {
         let driver = driver.clone();
+        let auction_id = summary.auction_id;
+        let settlement_id = summary.settlement_id;
```

Comment:
As a side note - is there a reason we need a separate `settlement_id` and `auction_id`? Is it because you expect the driver to have to keep track of multiple settlements per `auction_id`? If so, is it possible to keep this internal so the concept of a `settlement_id` doesn't "leak" into the `autopilot` (and whatever calls the driver?).

Reply:
I think you are right. As long as the `autopilot` only calls `/solve` once per auction for each solver there is no need for it to be aware of any id besides the `auction_id`. I will update the code to not let the `settlement_id` leak out.

Reply:
I remember why I added that. The logging logic in `submit_settlement()` expected a `settlement_id` for associating the logs to 1 of many settlements. Let's see how to best deal with that across the current and new driver.

Reply:
I ended up dropping the `settlement_id` and made `submit_settlment()` expect an `Option<u64>` for the id.

Reply:
>  expected a settlement_id for associating the logs to 1 of many settlements. Let's see how to best deal with that across the current and new driver.

Ah right - makes sense. I think its good that we remove this for the `autopilot` + `driver` as it shouldn't be needed anymore (or at least, the `auction_id` should be good enough).